server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/IndexItemNode.java

```diff
@@ -34,8 +36,8 @@ protected IndexItemNode(Locality location, String name, boolean global) {
   }
 
   @Override
-  public List<Node> getDefinitions() {
-    return ImmutableList.of(this);
+  public List<Location> getDefinitions() {
+    return ImmutableList.of(this).stream().map(Node::getLocality).map(Locality::toLocation).collect(Collectors.toList());
```

Comment:
Too complicated.
You can create ImmutableList from a location, not from `this`.